docs/src/main/sphinx/release/release-396.md

```diff
@@ -15,11 +15,18 @@
   the default threshold before the predicate is compacted. ({issue}`14029`)
 * Extend the range of possible values of the ClickHouse `Date` and `DateTime`
   types. ({issue}`11116`)
+* Fix failure when specifying a table or column comment that contains special
+  characters. ({issue}`14058`)
 
 ## Delta Lake connector
 
+* Add support for writing to tables using writer v3. This does not yet include
+  support for `CHECK` constraints. ({issue}`14068`)
+* Add support for reading tables with the table property
+  `delta.columnMapping.mode` set to `id`. ({issue}`13629`)
 * Improve performance when writing
   [structural data types](structural-data-types) to Parquet files. ({issue}`13714`)
+* Attempt to undo the operation when writing a checkpoint file fails. ({issue}`14108`)
```

Comment:
What's the impact of this? Or what was the problem before this was implemented?

Reply:
I think we were just ignoring the failures and proceeding as if they hadn't happened. I'm not sure what the downstream implications of that are.